### ISSUE TITLE
AdminPageChooser shows itself

### ISSUE THREAD

**Issue by salvadormrf** [torchbox/wagtail#1332]
The AdminPageChooser result list shows itself when:
- When searching (typing) for a page and pressing Enter key on search input.
- When searching for a page and removing all text.

This was happening to me, not sure if has been fixed.

**Comment by gasman** [comment]
Hi @salvadormrf,

I don't quite understand the problem you're reporting here - please can you rephrase this in terms of "expected behaviour" / "actual behaviour"?

**Comment by salvadormrf** [comment]
Tested on demo and was unable to reproduce this issues. 
Issue may be caused by my custom changes.
